index.js

```diff
@@ -425,6 +428,32 @@ var extensions = {
       },
     },
   ],
+  '.cts': [
+    'ts-node/register',
+    {
+      module: '@swc/register',
+      register: function (hook, config) {
+        config = config || {
+          only: [endsInCts],
+          ignore: [isNodeModules],
+          jsc: {
+            parser: {
+              syntax: 'typescript',
+            },
+          },
+          module: {
+            type: 'commonjs',
+          },
+        };
+
+        hook(
+          Object.assign({}, config, {
+            extensions: '.cts',
+          })
+        );
+      },
+    },
```

Comment:
Shall we drop this swc/register? I think we might as well make ts-node the only supporter of `.cts` unless someone asks for it.